CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.8.1
```

Comment:
Or 0.9.0 due to the new optional `display` arg? but I think either are fine.

Reply:
If we'd be over 1.x I'd definitely do a minor version bump, but it is different for `0.x` afaik projects that specify caret semver range like`^0.8.0` won't get minor updates, i.e `0.9.0`. But they would get minor updates if we'd be over `1.x`. So for this I'd do a patch, see also https://github.com/npm/node-semver#caret-ranges-123-025-004